integrated_channels/blackboard/transmitters/content_metadata.py

```diff
@@ -19,3 +19,9 @@ def __init__(self, enterprise_configuration, client=BlackboardAPIClient):
             enterprise_configuration=enterprise_configuration,
             client=client
         )
+
+    def _prepare_items_for_transmission(self, channel_metadata_items):
+        # here is a hack right now to send only one item
+        # we have to investigate how to handle multiple
+        # metadata items since there is no batch course create endpoint in blackboard
+        return channel_metadata_items[0]
```

Comment:
but what about chunk_size default of 1?

Reply:
ah yeah need to do that too thanks for the reminder...